Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -85,6 +95,18 @@ public void Generate()
 
         public void TearDown()
         {
+            var dotnet = _tools.GetTool(Dotnet);
+
+            const string dotnetToolsFileName = "dotnet-tools.json";
+            var hasDotnetToolManifest = Directory.GetFiles(SolutionDirectory, dotnetToolsFileName, SearchOption.AllDirectories)
```

Comment:
Move this into a new abstraction `IDotnetToolInstaller` with a method `InstallTool(string toolName, string? version)`

Reply:
Addressed in: [PR Add dotnet tool installer](https://github.com/messerli-informatik-ag/meta-generator/pull/24)